### PR TITLE
JS -- Actions must be evaluated in global scope

### PR DESCRIPTION
**Pull request by calixteman**
* All the public properties of doc are injected into globalThis, in order to make them available through `this`
 * Put event in the global scope too.

### CODE REVIEW

File: src/scripting_api/initialization.js [hunk 1]
```diff
@@ -68,34 +71,57 @@ function initSandbox({ data, extra, out }) {
...
+  globalThis.zoomtype = ZoomType;
 
   for (const name of Object.getOwnPropertyNames(AForm.prototype)) {
     if (name !== "constructor" && !name.startsWith("_")) {
-      out[name] = aform[name].bind(aform);
+      globalThis[name] = aform[name].bind(aform);
+    }
+  }
+
+  // The doc properties must live in the global scope too
```
Comment (brendandahl) [inline src/scripting_api/initialization.js:106]: I'm guessing it's not possible to make the global scope be the document?

Comment (brendandahl) [reply src/scripting_api/initialization.js:106]: For example when I do `console.println(this)` in acrobat, i get `[object Doc]`

Comment (calixteman) [reply src/scripting_api/initialization.js:105]: I think that in a way this code change globalThis into a kind of doc.
About the second point, we can change the implementation of `toString`.
In chrome, it displays `[object global]`.

**Review by brendandahl — APPROVED** [review]

**Pull request merged**